### DISCUSSION TITLE
Create Uno Platform App (5.3.18.22) via command line?

### DISCUSSION THREAD

**Discussion by ward-horsfall** [unoplatform/uno#17812] [Q&A] [answered]
Hi,

Using the "donet new" command line what is the syntax to create a Uno platform app to
create an app with all the defaults selected as if you accepted all the defaults via
the wizard. What would it be?

Thanks

Ward

**Comment by jeromelaban** [comment]
Thank for the question!

You can find the syntax using the live wizard: https://new.platform.uno.

**Reply by francoistanguay** [reply, marked as answer]
You can also find an explanation of the various options here: https://platform.uno/docs/articles/getting-started/wizard/using-wizard.html
